R/convGDX2MIF.R

```diff
@@ -5,26 +5,28 @@
 #'
 #'
 #' @param gdx a GDX as created by readGDX, or the file name of a gdx
-#' @param gdx_ref reference-gdx for policy costs, a GDX as created by readGDX, or the file name of a gdx
+#' @param gdx_ref reference-gdx for < cm_startyear, used for fixing the prices to this scenario
```

Comment:
there is no `gdx_ref`argument anymore

Reply:
that is not correct, we still need `gdx_ref`…